docs/ProductImages.md

```diff
@@ -64,16 +64,30 @@ You should follow the Styles API instruction in the main [README](/README.md#sty
 
 Below, we describe the namespace that are defined in the `ProductImages`.
```

Comment:
Remove useless text
```suggestion
```

And change title to whatever @guerreirobeatriz say it's the right one